routes/web.php

```diff
@@ -39,3 +40,22 @@
 //View scenario route.
 Route::get('/stories/{story}/scenarios/{scenario}', [ScenarioController::class, 'show'])->name('scenario.show');
 
+//Route::get('/filter', '\App\Http\Controllers\StoryController@filter');
+Route::get('categories/{category:slug}', function (scopeFilter $category) {
+    return view('stories', [
+        'stories' => $category->stories,
+        'currentCategory' => $category,
+        'categories' => Category::all()
+    ]);
+})->name('category');
+
+//Search route.
+Route::get('/search', '\App\Http\Controllers\StoryController@search');
```

Comment:
hebben we deze route echt nodig? zoekt 'ie niet op de standaard index / stories pagina

Reply:
Inderdaad, heb m verwijderd